heron/common/src/java/com/twitter/heron/common/utils/misc/SerializeDeSerializeHelper.java

```diff
@@ -33,7 +36,10 @@ public static IPluggableSerializer getSerializer(Map<String, Object> config) {
     try {
       String serializerClassName = (String) config.get(Config.TOPOLOGY_SERIALIZER_CLASSNAME);
       if (serializerClassName == null) {
-        serializer = new KryoSerializer();
+        LOG.log(Level.WARNING, "Serializer class name not provided");
+        LOG.log(Level.WARNING, "Fall back to Java serializer. "
+                + "THIS COULD CAUSE SERIOUS PERFORMANCE DEGRADATION");
```

Comment:
No need for all caps, that's what warning is intended to do. Also can you specify in the log message what someone should do differently (i.e. that they should use Kryo and how to configure it). If we have a link to the documentation for how to use Kryo that would be ideal to include in the message.